src/handlers/comment/handlers/payout.ts

```diff
@@ -1,62 +1,66 @@
-  if (!issue) {
-    logger.info(`Skipping '/payout' because of no issue instance`);
-    return;
-  }
+//   const payload = _payload as Payload;
+//   logger.info(`Received '/payout' command from user: ${payload.sender.login}`);
+//   const issue = (_payload as Payload).issue;
+//   if (!issue) {
+//     logger.info(`Skipping '/payout' because of no issue instance`);
+//     return;
+//   }
 
-  const _labels = payload.issue?.labels;
-  if (_labels?.some((e) => e.name.toLowerCase() === "Permitted".toLowerCase())) {
-    logger.info(`Permit already generated for ${payload.issue?.number}`);
-    return;
-  }
+//   const _labels = payload.issue?.labels;
+//   if (_labels?.some((e) => e.name.toLowerCase() === "Permitted".toLowerCase())) {
+//     logger.info(`Permit already generated for ${payload.issue?.number}`);
+//     return;
+//   }
 
-  const IssueComments = await getAllIssueComments(issue.number);
-  if (IssueComments.length === 0) {
-    return `Permit generation failed due to internal GitHub Error`;
-  }
+//   const IssueComments = await getAllIssueComments(issue.number);
+//   if (IssueComments.length === 0) {
+//     return `Permit generation failed due to internal GitHub Error`;
+//   }
 
-  const hasPosted = IssueComments.find((e) => e.user.type === "Bot" && e.body.includes("https://pay.ubq.fi?claim"));
-  if (hasPosted) {
-    logger.info(`Permit already generated for ${payload.issue?.number}`);
-    return;
-  }
+//   const hasPosted = IssueComments.find((e) => e.user.type === "Bot" && e.body.includes("https://pay.ubq.fi?claim"));
+//   if (hasPosted) {
+//     logger.info(`Permit already generated for ${payload.issue?.number}`);
+//     return;
+//   }
 
-  // assign function incentivesCalculation to a variable
-  const calculateIncentives = await incentivesCalculation();
+//   // assign function incentivesCalculation to a variable
+//   const calculateIncentives = await incentivesCalculation();
 
-  const creatorReward = await calculateIssueCreatorReward(calculateIncentives);
-  const assigneeReward = await calculateIssueAssigneeReward(calculateIncentives);
-  const conversationRewards = await calculateIssueConversationReward(calculateIncentives);
-  const pullRequestReviewersReward = await calculatePullRequestReviewsReward(calculateIncentives);
+//   const creatorReward = await calculateIssueCreatorReward(calculateIncentives);
+//   const assigneeReward = await calculateIssueAssigneeReward(calculateIncentives);
+//   const conversationRewards = await calculateIssueConversationReward(calculateIncentives);
+//   const pullRequestReviewersReward = await calculateReviewContributorRewards(calculateIncentives);
 
-  return await handleIssueClosed(creatorReward, assigneeReward, conversationRewards, pullRequestReviewersReward, calculateIncentives);
-};
+//   return await handleIssueClosed(creatorReward, assigneeReward, conversationRewards, pullRequestReviewersReward, calculateIncentives);
+// }
```

Comment:
I'm unsure why I left this commented out. I have yet to see this used in production. Perhaps this was implemented incorrectly? Or else I'll need to review later.

Reply:
This was never called anywhere.